fastjet/internal/delaunay/point.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"math"
+	"strconv"
+)
+
+// Point in the X-Y Plane, that holds dynamic information
+type Point struct {
+	X       float64     // X coordinate
+	Y       float64     // Y coordinate
+	adjT    []*Triangle // holds all triangles adjacent to a point
+	outside bool        // true if point is on the outside of the triangulation
+	Ind     int         // user defined identifier
```

Comment:
I need the ID for two things: 

To remove points: When a point is removed copies of the adjacent points are made and a new triangulation with just those copies is started. After it ran through, the ID tells me which copy belongs to which point and the point can be updated accordingly. 

For the jet clustering: I will need to know which jet corresponds to which point